cpp/serve/engine.cc

```diff
@@ -100,6 +98,17 @@ class EngineImpl : public Engine {
                      engine_config->additional_model_lib_paths[i], /*model_index=*/i + 1);
     }
 
+    // Step 3. Initialize tokenizer and grammar
+    this->tokenizer_ = Tokenizer::FromPath(engine_config->model);
+    ICHECK(model_configs[0].count("token_table_postproc_method") > 0)
```

Comment:
Let us make things backward compatible, if not found, pick an old default for now